modules/@apostrophecms/page/index.js

```diff
@@ -2108,7 +2123,10 @@ database.`);
               _id: null
             });
           } else {
-            query.project(self.options.publicApiProjection);
+            query.project({
+              ...self.options.publicApiProjection,
+              cacheInvalidatedAt: 1
```

Comment:
We need to have this value in order to [construct the ETag](https://github.com/apostrophecms/apostrophe/pull/3692/files#diff-0f2d52f6b9cfe649f1012ea4dc4af8bbdd7a0b8cba9db6a9748729b51e8e76d6R478) value, but that's not ideal to always expose it....

I'm open to suggestions!

Reply:
Just expose it, we add some other things preemptively because we know the user needs them for success. We could delete it afterwards but it would be slow and bug-prone.